docs/fundamentals/code-analysis/style-rules/naming-rules.md

```diff
@@ -19,29 +19,29 @@ helpviewer_keywords:
 ---
 # Code-style naming rules
 
-In your `.editorconfig` file, you can define **naming rules** that specify and enforce how .NET programming language code elements&mdash;such as classes, properties, and methods&mdash;should be named. For example, you can specify that public members must be capitalized, or that private fields must begin with `_`.
+In your `.editorconfig` file, you can define naming conventions for your .NET programming language code elements&mdash;such as classes, properties, and methods&mdash;and how the compiler or IDE should enforce those conventions. For example, you could specify that a public member that isn't capitalized should be treated as a compiler error, or that if a private field doesn't begin with an `_` a build warning should be issued.
```

Comment:
```suggestion
In your `.editorconfig` file, you can define naming conventions for your .NET programming language code elements&mdash;such as classes, properties, and methods&mdash;and how the compiler or IDE should enforce those conventions. For example, you could specify that a public member that isn't capitalized should be treated as a compiler error, or that if a private field doesn't begin with an `_`, a build warning should be issued.
```

Reply:
Does it make sense to use `*` `*` which is rendered as italics? "`.editorconfig`" is the literal file name.